src/csharp/Grpc.Core/Grpc.Core.csproj

```diff
@@ -8,7 +8,7 @@
     <AssemblyTitle>gRPC C# Core</AssemblyTitle>
     <VersionPrefix>$(GrpcCsharpVersion)</VersionPrefix>
     <Authors>Google Inc.</Authors>
-    <TargetFrameworks>net45;netstandard1.5</TargetFrameworks>
+    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
```

Comment:
I don't think we want to start requiring netstandard2.0 just for the sake of adding Arm detection support.

Reply:
![grafik](https://user-images.githubusercontent.com/16267227/47567508-08da1380-d92f-11e8-8a62-c20d9f439aa0.png)

Reply:
If someone still uses .NET core 1.x  and  Grpc.Core suddenly becomes `netstandard2.0` (i.e the `netstandard1.5` directory with the Grpc.Core.dll  assembly disappears from the nuget package and `netstandard2.0` will be add), I'm pretty sure that will break the users. You can't run 2.0 libraries on older .NET core installation. 

I think the article you highlighted just says that if you have a netstandard1.5 library, you can run it on .NET core 2.x which supports netstandard2.0 and it will work (which makes sense), it doesn't mean that netstandard2.0 libraries (which require more APIs) will magically work on older .NET core installations (which AFAIK is not the case).